### ISSUE TITLE
acme.sh always generates a log file even without '--log' option

### ISSUE THREAD

**Issue by Panblack** [Neilpang/acme.sh#1861]
The default logfile name is based on LOG_FILE variable in account.conf .
This could be an issue when a user does not want to leave an log file withou even konwing it.

**Comment by kobruleht** [comment]
Specify /dev/null  as log file name

**Comment by Neilpang** [comment]
edit `~/.acme.sh/account.conf`

remove the `LOG_FILE=xxxxx` line.

**Comment by Panblack** [comment]
> edit `~/.acme.sh/account.conf`
> 
> remove the `LOG_FILE=xxxxx` line.

Yes, I did that in my script. But it will be better if the the `LOG_FILE=xxxxx` line does not appear in account.conf automatically unless manually configured. 

Best wishes.

**Comment by Panblack** [comment]
> Specify /dev/null as log file name

Thanks. I need the log when renewing certs, while I do not need it when listing certs. ;)

**Comment by neilpang** [comment]
Closing as stale: no activity for over 18 months. If this still applies on the current release, please open a new issue with the `--debug 2` log.

**Comment by neilpang** [comment]
Not a bug: LOG_FILE is only persisted to account.conf when --log is used; remove the LOG_FILE line to disable logging.
